third_party/nvfuser/csrc/ir_nodes.cpp

```diff
@@ -2062,6 +2133,12 @@ std::pair<IterDomain*, IterDomain*> IterDomain::split(
     Val* factor,
     bool inner_split,
     bool trim_out_of_bounds) {
+  VAL_LOG_EXPLICIT(
+      in,
+      "IterDomain::split",
+      factor->toString(0, SerializationFormat::NameOnly),
+      std::to_string(inner_split),
+      std::to_string(trim_out_of_bounds), );
```

Comment:
There's `Printer` utility class in utils.h. Could it help reduce repetition? For example, could `VAL_LOG_EXPLICIT` just take whatever objects and use `Printer` to covert to a string?